src/app/page.tsx

```diff
@@ -13,6 +14,7 @@ export default async function Home() {
 
   return (
     <>
+      <ProgressBar />
```

Comment:
can't we use the framer motion here? The same is implemented in:
`src/components/pages/contribute/index.tsx`